components/admin/actions.tsx

```diff
@@ -341,94 +134,138 @@ export default function Actions({ data }: PageProps): JSX.Element {
         values.biddingNumberType === 'generate';
     }
 
-    updateApplication(request);
-    setTimeout(() => router.reload(), 500);
+    try {
+      updateApplication(request).then(() => {
+        router.reload();
+      });
+    } catch (error) {
+      console.log('error', error);
```

Comment:
what's the benefit of catching this error? For context I'm setting up Sentry monitoring at the moment which will notice when errors get thrown and record them centrally.

Reply:
Will remove